x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/dataframe/evaluation/regression/Regression.java

```diff
@@ -76,7 +76,7 @@ public Regression(String actualField, String predictedField, @Nullable List<Eval
     }
 
     private static List<EvaluationMetric> defaultMetrics() {
-        return Arrays.asList(new MeanSquaredError(), new RSquared());
+        return Arrays.asList(new MeanSquaredError(), new RSquared(), new Huber());
```

Comment:
Was it on purpose not to include MSLE here? Is it because we cannot set `offset` automagically for the user?

Reply:
It was intentional indeed. The reason is because we cannot calculate MSLE for negative values.